src/components/Users/UserAddEditForm.tsx

```
@@ -0,0 +1,1200 @@
+                                <CareIcon
+                                  icon="l-check-circle"
+                                  className="text-xl text-green-500"
+                                />{" "}
+                                <span className="text-primary-500">
+                                  {t("username_available")}
+                                </span>
+                              </div>
+                            )}
+                          </>
+                        )}
+                      </>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)